packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -194,7 +194,7 @@ export class InternalStateProvider implements CSP.IChainStateService {
       if (found.blockHeight && found.blockHeight >= 0) {
         confirmations = tipHeight - found.blockHeight + 1;
       }
-      const convertedTx = TransactionModel._apiTransform(found, { object: true }) as Partial<ITransaction>;
+      const convertedTx = TransactionModel._apiTransform(found, { object: true }) as TransactionJSON;
```

Comment:
This is annoying. Need to get conditional types for the return types of _apiTransform